Source/WebCore/rendering/RenderBox.h

```diff
@@ -668,6 +668,7 @@ override;
 
     bool shouldComputeLogicalHeightFromAspectRatio() const;
 
+    bool shouldIgnoreMinMaxSizes() const;
```

Comment:
Style nit: please add an empty line after the method.

Reply:
Fixed